### PR TITLE
Oracle SSL testing

### PR DESCRIPTION
**Pull request by KyleAure**
Introduce test of Oracle using SSL.
Continuing work started in #15109

Useful reference docs for configuring Oracle container with SSL:
- https://oracle-base.com/articles/misc/configure-tcpip-with-ssl-and-tls-for-database-connections
- https://stackoverflow.com/questions/41210109/jdbc-connect-to-oracle-database-with-ssl
- https://www.oracle.com/technetwork/database/enterprise-edition/wp-oracle-jdbc-thin-ssl-130128.pdf

### CODE REVIEW

**Comment by KyleAure** [comment]
#build #libbybot

File: dev/com.ibm.ws.jdbc_fat_oracle/publish/servers/com.ibm.ws.jdbc.fat.oracle.ssl/bootstrap.properties [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+com.ibm.ws.logging.trace.specification=*=info:RRA=all:oracle.*=all:oracle.*=all
```
Comment (nmittles) [inline dev/com.ibm.ws.jdbc_fat_oracle/publish/servers/com.ibm.ws.jdbc.fat.oracle.ssl/bootstrap.properties:11]: Duplicate entry of `oracle.*=all`

**Review by nmittles — APPROVED** [review]

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
#libbybot

**Pull request merged**